Examples/AD/AD Application Group Add From List.ps1

```diff
@@ -0,0 +1,95 @@
+#Requires -Version 7.0
+
+<#
+.SYNOPSIS
+A sample script to query a deviceapplication Dashworks list and add devices to application groups.
+
+.DESCRIPTION
+A sample script to query a device application Dashworks list, look up the ad group related to the application
+from a taskID and add the device to the group in question.
+The script is designed to be run on a server ActiveDirectory RSAT powershell module installed.
+The list used must contain the "app key" column. This is required to get the task values for that objects.
+In normal usage the list used should contain a filter to exclude the success and failed values of the task
+for devices to be updated. For example, the list might pick up the "Not Started" values in an
+NNSFC created task, once these have been changed to Complete or Failed, they will no longer
+appear in the task
+#>
+
+[CmdletBinding()]
+param (
+[Parameter(Mandatory=$true)]
+[string]$DwInstance,
+[Parameter(Mandatory=$true)]
+[string]$DwAPIKey,
+[Parameter(Mandatory=$true)]
+[int]$ListID,
+[Parameter(Mandatory=$true)]
+[int]$ProjectID,
+[Parameter(Mandatory=$true)]
+[int]$TaskID,
+[Parameter(Mandatory = $true)]
+[ValidateSet("Device", "User")]
+[string]$ObjectType
+)
+
+$ObjectTypeApplication = switch($ObjectType) {
+    "Device" {"deviceapplications"}
+    "User"   {"userapplications"}
+}
+
+$DwInstance = 'https://master.internal.juriba.com:8443'
+$DwAPIKey = 'ppk+RarkCC966ghZkMxr1uT3uMSphnt+e+rjV8Q+ATDtnBjFvcHHw/BK1c9Jqa7tCUEfPApKiGLwDmpvMiRUyw=='
+$listID = 5499
+$ProjectID = 85
+$AppGrpTaskID = 14051
+$ObjectType = 'Application'
+$List = Export-DwList -Instance $DwInstance -APIKey $DwAPIKey -ListId $listID -ObjectType $ObjectTypeApplication
+
+$AppKeys | foreach-Object -Parallel {
```

Comment:
Where does $AppKeys come from?

Reply:
I've checked in the line that got clipped during testing:

$AppKeys = (($List | select-object -Property "Application Key" -Unique).'Application Key')